packages/framework/client-logger/external-telemetry/src/common/telemetry/index.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { ContainerTelemetryEventName } from "../../container/index.js";
+
+/**
+ * The base interface extended by all external telemetry
```

Comment:
Since this is a core interface in the API, it would be good to offer a bit more detail in the docs here. Maybe even an `@example` block.